cdap-app-templates/cdap-etl/cdap-etl-core/src/main/java/co/cask/cdap/etl/common/Constants.java

```diff
@@ -64,6 +64,14 @@ private Transform() {
   }
 
   /**
+   * Constants related to Connections
+   */
+  public static final class Connections {
+    public static final String PLUGINID = "connections";
```

Comment:
this is a weird name for this constant. It's used as the key to store the connections, but not related much to plugin ids.  Maybe just have a top level constant called CONNECTIONS.